load_dumps.php

```diff
@@ -0,0 +1,25 @@
+<?php
+require('app/config.php');
+
+$db = new PDO('mysql:dbname='.DB_NAME, DB_USER, DB_PASS);
+
+$sql = file_get_contents('databases/saathi.sql');
+$qr = $db->exec($sql);
```

Comment:
Script does not give any error even if same insert operation is run again.
So error checking is not done. Removed extra variable name $qr.